### PR TITLE
feature/corporation-page

### PR DESCRIPTION
**Pull request by cricar6**
Screen Shot 2021-09-29 at 12 23 09 AM
Screen Shot 2021-09-29 at 12 23 04 AM
Screen Shot 2021-09-28 at 7 00 27 PM

### Updates and additions
Big Image Layout Updated to support an array of text, and images on body-text for "Somos Acción y Participación" and "Nuestros Ejes de Incidencia"
Last Projects updated to fit Big Image Layout new Structure
Corporation Page created with all components working properly
Member Page "Quienes lo hacen posible" section created properly with responsive and desktop resolutions
Mixins for other pages related with BigImageLayout and their structure updated

### Observations
There are repeated keys on Member section due to repeated people: "Andres Marin", "Felipe Salcedo"

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,7 +21,8 @@
     ]
   },
   "dependencies": {
-    "@contentful/rich-text-react-renderer": "^15.1.0",
+    "@contentful/rich-text-react-renderer": "^15.4.0",
+    "@contentful/rich-text-types": "^15.3.6",
```
Comment (tatarodriguez11) [inline package.json:25]: porque actualizaste la version de las dependencias?

**Review by javiermlovo — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (cricar6) [reply package.json:25]: Porque era necesario actualizarlas para que pudieran leer assets embebidos dentro de big-image-layout

**Review by juanbravozu — APPROVED** [review]

File: components/BigImageLayout.js [hunk 2]
```diff
@@ -1,15 +1,33 @@
...
+import EmbeddedAssetImage from './EmbeddedAssetImage';
 
-export default function BigImageLayout({ contentType, data, variation }) {
-  function getContent(innerContent, innerContentType) {
-    if (innerContentType === 'rich') {
-      return documentToReactComponents(innerContent);
-    } if (innerContentType === 'simple') {
-      return <p>{ innerContent }</p>;
+export default function BigImageLayout({ data, variation }) {
+  function generateElement(type, content) {
```
Comment (Cruzlex7) [inline components/BigImageLayout.js:8]: always is a paragraph

Comment (Cruzlex7) [reply components/BigImageLayout.js:8]: must be a separated function.

File: components/EmbeddedAssetImage.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+export default function EmbeddedAssetImage({ url }) {
+  return (
+    <>
+      <img
```
Comment (Cruzlex7) [inline components/EmbeddedAssetImage.js:6]: Use it directly.

File: components/BigImageLayout.js [hunk 4]
```diff
@@ -1,15 +1,33 @@
...
+  const options = {
+    renderNode: {
+      [BLOCKS.EMBEDDED_ASSET]: (node) => {
+        const { file: { url } } = node.data.target.fields;
+        return <EmbeddedAssetImage url={url} />;
+      },
+    },
+  };
+
+  function getContent(innerContent) {
```
Comment (Cruzlex7) [inline components/BigImageLayout.js:22]: this function could be outside, receiving 'innerContent' and 'options'

File: components/MembersGrid.js [hunk 5]
```diff
@@ -0,0 +1,45 @@
+export default function MembersGrid({ fields }) {
```
Comment (Cruzlex7) [inline components/MembersGrid.js:1]: use destructuring for fields props.

File: components/MembersGrid.js [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+          <li key={id}>
+            <article className="member">
+              <img
+                alt=""
+                src={`https://${imageUrl}`}
+                className="member__image"
+              />
+              <div>
+                <p className="member__name">
+                  {' '}
```
Comment (Cruzlex7) [inline components/MembersGrid.js:34]: spacing.

**Review by Cruzlex7 — APPROVED** [review]

**Review by javiermlovo — APPROVED** [review]

**Review by juanbravozu — APPROVED** [review]

**Pull request merged**